dockstore-webservice/src/test/java/io/dockstore/webservice/languages/WDLHandlerTest.java

```diff
@@ -0,0 +1,69 @@
+package io.dockstore.webservice.languages;
+
+import java.util.Collections;
+
+import io.dockstore.webservice.core.Workflow;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class WDLHandlerTest {
+
+    private static final String GOOD_WDL =
```

Comment:
I feel like it might be better to just read these from files instead

Reply:
I agree; will change.